packages/terra-clinical-header/lib/Header.scss

```diff
@@ -0,0 +1,22 @@
+@import '~terra-mixins';
+
+.terraClinical-Header {
+  background-color: $terra-grey-5;
+  border-bottom-color: $terra-grey-15;
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  padding: 0.714rem;
+  width: 100%;
+
+  .terraClinical-Header--title {
```

Comment:
Is title intended to be a modifier class? or should it be .terraClinical-Header-title?